reporter/src/main/kotlin/utils/SpdxDocumentModelMapper.kt

```diff
@@ -211,6 +215,25 @@ private fun ScanResult.toSpdxPackageVerificationCode(): SpdxPackageVerificationC
         packageVerificationCodeValue = summary.packageVerificationCode.toLowerCase()
     )
 
+private fun SpdxDocument.addExtractedLicenseInfo(licenseTextProvider: LicenseTextProvider): SpdxDocument {
+    val nonSpdxLicenses = packages.flatMapTo(mutableSetOf()) {
+        SpdxExpression.parse(it.licenseConcluded).licenses() + SpdxExpression.parse(it.licenseDeclared).licenses()
+    }.filter {
+        !SpdxConstants.isNotPresent(it) && SpdxLicense.forId(it) == null && SpdxLicenseException.forId(it) == null
```

Comment:
Nit: That initial double negation is not so super-ice. Should be spend an `isPresent()` convenience function in `SpdxConstants`?

Reply:
Done.

Reply:
I guess this was not pushed yet?

Reply:
Now it is.